### PR TITLE
Dev tools and page layout

### PR DESCRIPTION
**Pull request by cszczepaniak**
# What I added (link any issues addressed)
- installed prettier, eslint, lint-staged, and husky to format and lint our code whenever we commit
  - you can bypass this by passing `-n` when you commit but avoid this so our code formatting stays consistent
- set up an initial page layout (app bar and page container). it can completely change if we want it to, but I wanted to get something going 

# How to test it
- first, `npm install`
- for the dev tools, you can try creating lint warnings (create an unused variable is an easy one) and try committing. you should get yelled at and your change shouldn't be committed
- for the page layout, start the app and look at the page. click on the "my team" link and see how that looks - the app bar and the padding/spacing/everything else is in PageWrapper, so it should stay consistent across pages

### CODE REVIEW

**Comment by adamszcz** [comment]
It let me commit this. Do I need to actually configure the rule before testing?

```
    useEffect(() => {
        const postLoginToBackend = async () => {
            const res = await axios.post('/api/player/login', {
                email: user?.email,
                name: user?.displayName,
            });
            **var a;**
            setIsReturning(res.data.isReturning);
        };
        postLoginToBackend();
    });
```

**Comment by cszczepaniak** [comment]
> It let me commit this. Do I need to actually configure the rule before testing?
> 
> 
> 
> ```
> 
>     useEffect(() => {
> 
>         const postLoginToBackend = async () => {
> 
>             const res = await axios.post('/api/player/login', {
> 
>                 email: user?.email,
> 
>                 name: user?.displayName,
> 
>             });
> 
>             **var a;**
> 
>             setIsReturning(res.data.isReturning);
> 
>         };
> 
>         postLoginToBackend();
> 
>     });
> 
> ```

I forgot a step which I've added above... you need to `npm install` first so I suspect that's your problem.

**Comment by adamszcz** [comment]
Nvm, got it to work!

**Review by adamszcz — APPROVED** [review]
I will need to read up on linting more to understand it, but I got this working on my end with no issues. Husky threw up an error when I tried to commit with an unused variable

**Comment by cszczepaniak** [comment]
> I will need to read up on linting more to understand it, but I got this working on my end with no issues. Husky threw up an error when I tried to commit with an unused variable 

TL;DR - linting enforces rules that aren't compiler errors but also aren't good things to have in your code. It can be completely configured but it will catch things like unused variables, unused function arguments, under specified types, etc.
